modules/installation-and-upgrade/pages/install-proxy-vm.adoc

```diff
@@ -0,0 +1,133 @@
+| VirtIO Disk 2       | 100 GB for [path]``/var/cache/``
+| VirtIO Disk 3       | 101 GB for [path]``/srv/``
+| Name:               | suse-manager-proxy-setup
+| Network             | Bridge _br0_
+|===
+
+[NOTE]
+====
+For more information on {sle} Virtualization, see https://documentation.suse.com/sles/15-SP4/html/SLES-all/book-virtualization.html.
+====
+
+
+
+[[proxyvm.kvm.install]]
+== Install {productname} Proxy Virtual Machine
+
+
+Create virtual machine with three additional virtual disks required for the {productname} Proxy storage partitions.
+
+// FIXME: add details on networking (bridge).
+
+.Procedure: Creating the VM and Additional Partitions with KVM
+. Create a new virtual machine using the downloaded {productname} Proxy KVM image and select [guimenu]``Import existing disk image``.
+. Set [literal]``SUSE Linux Enterprise 15 SP4`` as the installed operating system.
+. Configure RAM and number of CPUs (at least 8 GB RAM and 2 CPUs).
+. Configure networking (bridge).
+. Name your KVM machine and select the [guimenu]``Customize configuration before install`` check box.
+. Click btn:[Add Hardware] to create additional virtual disks.
+  These disks will be partitioned and mounted later.
++
+
+[NOTE]
+====
+Storage size values are the absolute minimum—only suitable for a small test or demo installation.
+Also consider to create a separate partition for [path]``/srv/tftpboot/`` where images on Retail Branchservers are stored.
+====
++
+
+[cols="1,1,1", options="header"]
+|===
+| VirtIO Storage Disks | Name       | Sizing
+| VirtIO Disk 2        | squidcache | 101{nbsp}GB
+| VirtIO Disk 3        | tftpboot   | 100{nbsp}GB
+|===
+
+
+. Click btn:[Begin Installation] to boot the new VM from the {productname} Proxy image.
+  Wait until the login prompt is presented.
+
+
+== Run JeOS Firstboot Setup
+
+. Confirm to start the setup.
+. Select system locale such as [literal]``en_US``.
+. Select keyboard layout such as [literal]``us``.
+. Check end user license agreement.
+. Select time zone such as [literal]``UTC``.
+. Enter and confirm root password.
+  Keep note of the root password.
+
+
+
+
+[[minimmal.susemgr.prep]]
+== Preparing virtual machine for {productname} Proxy
+
+Before starting, obtain a registration code for this image from SUSE Customer Center - https://scc.suse.com.
+
+
+[[proc.minimal.susemgrproxy.prep]]
+.Procedure: Preparing for {productname} Proxy run
+
+. Log in as `root`.
+
+. Register {productname} Proxy with SCC:
++
+
+----
+SUSEConnect -e <EMAIL_ADDRESS> -r <SUSE_MANAGER_PROXY_CODE>
+----
++
+
+// FIXME (just taken over from server image installation
+
+. Prepare {productname} Proxy storage:
+  [path]``suma-storage`` command automatically prepares and configures previously created external storage for use with the proxy.
+  In the following command the first parameter is the device for {productname} Proxy Squid cache, the second parameter is the device for Retail images.
++
+
+----
+suma-storage /dev/vdb /dev/vdc
+----
```

Comment:
This does not work like this.

`suma-storage` for proxy takes just one parameter and handles only squid cache. saltboot/tftpboot for retail is not handled by this tool.

Reply:
ok --I thought you'd quickly enhance `suma-storage` as freshly documented ;)

I've now changed the procedure to match the reality.  I used the 4.2 build successfully.  Thanks for your patience!